playground/bicep/BicepSample.AppHost/redis.module.bicep

```diff
@@ -1,38 +1,33 @@
-targetScope = 'resourceGroup'
-
-@description('')
+@description('The location for the resource(s) to be deployed.')
 param location string = resourceGroup().location
 
-@description('')
 param keyVaultName string
 
-
-resource keyVault_IeF8jZvXV 'Microsoft.KeyVault/vaults@2022-07-01' existing = {
-  name: keyVaultName
+resource keyVault 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
```

Comment:
Why the downgrade in API version (surprising) /cc @tg-msft

Reply:
Let me look into it.